src/renderers/dom/fiber/__tests__/ReactDOMFiber-test.js

```diff
@@ -1117,5 +1117,35 @@ describe('ReactDOMFiber', () => {
           'to empty a container.',
       );
     });
+
+    it('should render a text component with a text DOM node on the same document as the container', () => {
+      // 1. Create a new document through the use of iframe
+      // 2. Set up the spy to make asserts when a text component
+      //    is rendered inside the iframe container
+      var textContent = 'Hello world';
+      var iframe = document.createElement('iframe');
+      document.body.appendChild(iframe);
+      var iframeDocument = iframe.contentDocument;
+      iframeDocument.write(
+        '<!DOCTYPE html><html><head></head><body><div></div></body></html>',
+      );
+      iframeDocument.close();
+      var iframeContainer = iframeDocument.body.firstChild;
+
+      var actualDocument;
+      var textNode;
+      iframeContainer.appendChildBackup = iframeContainer.appendChild;
```

Comment:
nit: remove

Reply:
good catch - goober from a previous attempt at it.